ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -238,9 +246,8 @@ let test_get_nbd_info =
 
   let open OUnit in
   "test_get_nbd_info" >:::
-  [
-  (*"test_returns_correct_uris" >:: test_returns_correct_uris *)
-    "test_returns_empty_list_when_no_host_is_connected_to_sr" >:: test_returns_empty_list_when_no_host_is_connected_to_sr
+  [ "test_returns_correct_uris" >:: test_returns_correct_uris
```

Comment:
The element type in the list is longer a uri, so perhaps rename this?

Reply:
Good catch, forgot that one!